client/app/src/main/java/net/tbp/interval/ui/backup/Signup.java

```diff
@@ -0,0 +1,140 @@
+package net.tbp.interval.ui.backup;
+
+import android.app.Activity;
+import android.content.ContentValues;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.util.Log;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.inputmethod.InputMethodManager;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+
+import com.example.interval.R;
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+
+import net.tbp.interval.ui.reminder.ReminderProvider;
+
+public class Signup extends Activity {
+    private Button signinBtn;           // button for user to go sign up
+    private Button submitBtn;           // button for user to go submit
+    private FirebaseAuth mAuth;
+    private EditText email, password, confirmPassword;
```

Comment:
for now let's do username (ex: jd123) instead of email

Reply:
it is firebase and it force to user email